src/main/java/io/vertx/cassandra/impl/Util.java

```diff
@@ -29,7 +28,7 @@ public void onSuccess(T result) {
       public void onFailure(Throwable t) {
         vertxFuture.fail(t);
       }
-    }, vertxInternal.getEventLoopGroup());
+    }, command -> vertx.getOrCreateContext().runOnContext(v -> command.run()));
```

Comment:
You can't capture the context here, because you're on a Cassandra driver I/O thread. Invoke `getOrCreateContext` at the top of the method body.